### ISSUE TITLE
Сборник документации и файлов

### ISSUE THREAD

**Issue by avokado03** [avokado03/MobileList#9]
Егор, в комменты прикрепляй все образцы и другие файлы.

**Comment by gora5624** [comment]
Хорошо, соберу материал и прикреплю

**Comment by gora5624** [comment]
Так, тут ссылка на архив, в архиве макеты по папкам разбиты, есть пояснения. Если не понятно что - пиши. И ссылка на видео с ютуба в ЛС отправил, ибо информация закрытая. https://drive.google.com/file/d/1aNSG21NjtJ-gChrrTb0hPSYNpePySQhC/view?usp=sharing

**Comment by avokado03** [comment]
Ок, сейчас главное - CSV, а макеты тогда до 2го спринта. Напоминаю про папки с приоритетами и про унификацию.

**Comment by gora5624** [comment]
Всё там. Заказ CSV, унифицированные макеты для тестов, макеты с ошибками, приоритеты на дополнительные макеты

**Comment by gora5624** [comment]
Обновил архив с материалами. Заказ разделён отдельно на глянцевый и матовый 
https://drive.google.com/file/d/1Oww8glR7NdtRDk48rCUcPchtTmKCZDzB/view?usp=sharing

**Comment by avokado03** [comment]
![Untitled Diagram](https://user-images.githubusercontent.com/35204052/89519570-3f6fde00-d7e5-11ea-8c82-7227411db23f.png)

Схема на 1й спринт

**Comment by gora5624** [comment]
Схема верная, с условием существования стрелки "Да" от "Найдены все PDF?", которую сожрал ромб.

**Comment by avokado03** [comment]
Принято решение о проверке корректности заказа на наличие только матовых/глянцевых пленок
